pkg/aws/aws_client/role.go

```diff
@@ -292,3 +305,52 @@ func completeRolePolicyDocument(statement map[string]interface{}) (string, error
 	assumeRolePolicyDocument, err := json.Marshal(rolePolicyDocument)
 	return string(assumeRolePolicyDocument), err
 }
+
+func (client *AWSClient) AttachPolicy(roleName string, policyArn string) error {
+	policyAttach := iam.AttachRolePolicyInput{
+		PolicyArn: &policyArn,
+		RoleName:  &roleName,
+	}
+	_, err := client.IamClient.AttachRolePolicy(context.TODO(), &policyAttach)
+	if err != nil {
+		return err
+	}
+	timeout := 2
+	start := 0
+
+	for start < timeout {
+
+		if attached, _ := client.PolicyAttachedToRole(roleName, policyArn); attached {
```

Comment:
You should return the `error` from this call that is currently assigned to `_` if it happens.